sickbeard/name_parser/regexes.py

```diff
@@ -426,6 +426,11 @@
      r'''
      ^(?P<ep_ab_num>\d{3,4})(-(?P<extra_ab_ep_num>\d{3,4}))?\.\s+(?P<series_name>.+?)\s-\s.*
      '''),
+     
+    ('anime_MangaOff',
+     # 003. Show Name - Ep Name.ext
+     # 003-004. Show Name - Ep Name.ext
+     ur'(?P<series_name>.+?)[ ._-]+(E(?P<ep_ab_num>\d{1,4})[ ._-]+)(.*)FANSUB[ ._-]+VOSTFR(.*?)([ ._-](\d{1,4}[pi])[ ._-])?(.*)[ ._-](?P<release_group>.*)'),     
```

Comment:
Well, i'm far from a regex expert, so i just wanted to catch some shitty release name on mangaoff.cc without risking to mess with the rest.